Frontend/library/package.json

```
@@ -0,0 +1,29 @@
+{
+    "name": "@epicgames/pixelstreamingfrontend",
+    "version": "0.0.1",
+    "description": "Frontend library for Pixel Streaming",
+    "main": "dist/pixelstreamingfrontend.min.js",
+    "types": "types/pixelstreamingfrontend.d.ts",
+    "scripts": {
+        "compile": "tsc --build --clean && tsc",
+        "build": "npm run build-webpack",
+        "build-webpack": "npx webpack",
+        "lint": "eslint . --ext .js,.jsx,.ts,.tsx",
+        "spellcheck": "cspell \"{README.md,.github/*.md,src/**/*.ts}\""
+    },
+    "devDependencies": {
+        "@typescript-eslint/eslint-plugin": "^5.16.0",
+        "@typescript-eslint/parser": "^5.16.0",
+        "cspell": "^4.1.0",
+        "eslint": "^8.11.0",
+        "prettier": "2.8.3",
+        "ts-loader": "^9.4.2",
+        "typedoc": "^0.23.24",
+        "webpack-cli": "^5.0.1"
```

Needs to add `webpack` and `typescript` as a devDependency. Tested to build this, and got an error about missing `webpack` and `typescript` if you don't have them installed globally.

Webpack-cli doesn't pull webpack as a transitive dependency since it's marked as a peerDependency, that needs to be provided by the project: https://github.com/webpack/webpack-cli/blob/master/package.json#L53

Tested and verified that the build passes with these installed as devDependencies.

```suggestion
        "typescript": "^4.9.4",
        "webpack": "^5.75.0",
        "webpack-cli": "^5.0.1"
```